ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master.yaml

```diff
@@ -54,13 +54,9 @@ tests:
   steps:
     cluster_profile: equinix-ocp-metal
     env:
-      DEVSCRIPTS_CONFIG: |
-        IP_STACK=v4
-        NETWORK_TYPE=OpenShiftSDN
+      CLUSTERTYPE: baremetal
```

Comment:
Adding baremetal here makes the job use a baremetal env instead of a virtual one, we don't have enough baremetal envs to do this can you remove it.

Reply:
Thanks @derekhiggins, now seeing upgrades went through, but tests seem to failing due to default step timeout 4h.  Let's see how it runs now with increased step timeout (1fe9244a5ba702bde582e62a358b0dac3fb02ad7) to 6h.